politeiad/plugins/comments/comments.go

```diff
@@ -50,6 +50,14 @@ const (
 	// SettingKeyTimestampsPageSize is the plugin setting key for the
 	// SettingTimestampsPageSize plugin setting.
 	SettingKeyTimestampsPageSize = "timestampspagesize"
+
+	// SettingKeyAllowEdits is the plugin setting key for the
+	// SettingAllowEdits plugin setting.
+	SettingKeyAllowEdits = "allowedits"
+
+	// SettingKeyEditPeriodTime is the plugin setting key for the
+	// SettingEditPeriodTime plugin setting.
+	SettingKeyEditPeriodTime = "editperiodtime"
```

Comment:
Suggestion, `editperiod` would be cleaner imo.  Up to you though.

Reply:
agreed